actions/bumper/entrypoint.sh

```diff
@@ -16,7 +16,7 @@ version_suffix=${VERSION_SUFFIX}
 hotfix_version=${HOTFIX_VERSION}
 
 cd ${GITHUB_WORKSPACE}/${source}
-
+git config --global --add safe.directory /github/workspace
```

Comment:
```suggestion
git config --global --add safe.directory $(pwd)
```

Reply:
Added the `| true` in case the command errors somehow, we don't want it to crash if there's an older git version or something